cli.js

```diff
@@ -34,6 +36,20 @@ files.forEach((file) => {
     } else {
       fs.writeFileSync(file, sorted, 'utf8')
       console.log(`${file} is sorted!`)
+      const diff = diffLines(packageJson, sorted)
+      diff.forEach((part) => {
+        const partLineList = part.value.split('\n')
```

Comment:
I think the EOL can be `\r` or `\r\n`

Reply:
I see.
Thank you for the information on how to respond.